code/modules/research/mechfab_designs.dm

```diff
@@ -111,57 +111,63 @@
 	req_tech = list(TECH_BIO = 3, TECH_MATERIAL = 3)
 
 /datum/design/item/mechfab/modules/armblade
-	name = "embed handblade"
+	name = "Embedded armblade"
 	build_path = /obj/item/organ_module/active/simple/armblade
 	id = "organ_mod_armblade"
 	materials = list(MATERIAL_STEEL = 16)
 	req_tech = list(TECH_BIO = 3, TECH_MATERIAL = 3)
 
 /datum/design/item/mechfab/modules/runner
-	name = "Mechanical muscules"
-	build_path = /obj/item/organ_module/muscule
+	name = "Mechanical muscles"
+	build_path = /obj/item/organ_module/muscle
 	id = "organ_mod_runner"
 	req_tech = list(TECH_BIO = 4)
 
 
 /datum/design/item/mechfab/modules/multitool
 	materials = list(MATERIAL_STEEL = 5)
 
-/datum/design/item/mechfab/modules/multitool/surgial
-	build_path = /obj/item/organ_module/active/multitool/surgial
-	name = "Surgial organ module"
-	id = "organ_mod_surgial"
+/datum/design/item/mechfab/modules/multitool/surgical
+	build_path = /obj/item/organ_module/active/multitool/surgical
+	name = "Embedded surgical multitool"
+	id = "organ_mod_surgical"
 	req_tech = list(TECH_BIO = 4)
 
 /datum/design/item/mechfab/modules/multitool/engineer
 	build_path = /obj/item/organ_module/active/multitool/engineer
-	name = "Technomancer organ module"
+	name = "Embedded Technomancer multitool"
 	id = "organ_mod_eng"
 	req_tech = list(TECH_BIO = 3, TECH_ENGINEERING = 3)
 
+/datum/design/item/mechfab/modules/multitool/miner
+	build_path = /obj/item/organ_module/active/multitool/miner
+	name = "Embedded mining multitool"
+	id = "organ_mod_min"
+	req_tech = list(TECH_BIO = 3, TECH_ENGINEERING = 3, TECH_MATERIAL = 4)
+
 //Prosthesis ====================================
 
 /datum/design/item/mechfab/prosthesis
 	category = "Prosthesis"
 	materials = list(MATERIAL_STEEL = 18)
 
 /datum/design/item/mechfab/prosthesis/r_arm
-	name = "right arm"
+	name = "Right arm"
```

Comment:
Aren't these improper nouns? How does this show up in game?

Reply:
Those are design datums, not items. They only show in HTML/Nano UIs.

Reply:
Oh I see.